queue/src/test/java/uk/gov/service/payments/commons/queue/sqs/SqsQueueServiceTest.java

```diff
@@ -62,7 +63,9 @@ public void setUp() {
     public void shouldSendMessageToQueueSuccessfully() throws QueueException {
```

Comment:
Could we have a unit test for the new overloaded method with the `delayInSeconds` parameter to ensure we're constructing the `SendMessageRequest` correctly with the delay set.

Reply:
thanks, now added.